### PR TITLE
Renamed amp_analytics_id to amp_gtm_id

### PR DESCRIPTION
**Pull request by ZhengC1**
Low hanging fruit issue #405 for a variable rename

### CODE REVIEW

**Comment by ZhengC1** [comment]
I signed it!

File: app/resources/admin.html.template [hunk 1]
```diff
@@ -1077,10 +1077,10 @@
       </div>
     </div>
     <div class="config">
-      <label for="amp_analytics_id">AMP Analytics ID (optional):</label>
+      <label for="amp_gtm_id">AMP Analytics ID (optional):</label>
```
Comment (gimite) [inline app/resources/admin.html.template]: Can you rename the label to "AMP GTM ID" as well?

File: app/resources/static/home.html.template:ja [hunk 2]
```diff
@@ -1,82 +0,0 @@
-{% extends "static-base.html.template" %}
```
Comment (gimite) [inline app/resources/static/home.html.template:ja]: Why did you delete these files?

File: app/resources/admin.html.template (hunk 1 above)
Comment (ZhengC1) [reply app/resources/admin.html.template]: Yes, sorry about that.

File: app/resources/static/home.html.template:ja (hunk 2 above)
Comment (ZhengC1) [reply app/resources/static/home.html.template:ja]: Will fix, i think the branch i checked out from must have been different from master.

File: app/main.py [hunk 3]
```diff
@@ -313,7 +313,7 @@ def setup_env(request):
     env.config = config.Configuration(env.repo or '*')
 
     env.analytics_id = config.get('analytics_id')
-    env.amp_analytics_id = config.get('amp_analytics_id')
+    env.amp_gtm_id= config.get('amp_gtm_id')
```
Comment (gimite) [inline app/main.py]: Insert a whitespace before `=`.

**Comment by ZhengC1** [comment]
i added a white space

**Comment by ZhengC1** [comment]
added white space.

**Comment by gimite** [comment]
Thanks! Merged. 

Looks like one of your commit: https://github.com/google/personfinder/pull/409/commits/f44cd2b620c3c3cbfdcaa69bf391e38a348a22bb has no author email address (that's why googlebot is complaining). Make sure to follow steps here:
https://help.github.com/articles/setting-your-commit-email-address-in-git/
to set your commit email address to avoid the issue in the future.
